docs/configuration/animations.md

```diff
@@ -11,11 +11,31 @@ The following animation options are available. The global options for are define
 | `duration` | `number` | `1000` | The number of milliseconds an animation takes.
 | `easing` | `string` | `'easeOutQuart'` | Easing function to use. [more...](#easing)
 | `onProgress` | `function` | `null` | Callback called on each step of an animation. [more...](#animation-callbacks)
-| `onComplete` | `function` | `null` | Callback called at the end of an animation. [more...](#animation-callbacks)
+| `onComplete` | `function` | `null` | Callback called when all animations are completed. [more...](#animation-callbacks)
+| `delay` | `number` | `undefined` | Delay before starting the animations.
```

Comment:
It's unclear to me if this would happen just the first time or before each iteration if looping is enabled. That could be worth clarifying if we want to keep the `loop` option

Reply:
First time only. Looping animation too has only one start time.